Sources/SwiftBasicFormat/generated/Format.swift

```diff
@@ -35,1865 +35,1865 @@ extension Format {
     }
     return result
   }
-  func format(syntax: UnexpectedNodesSyntax) -> UnexpectedNodesSyntax {
+  public func format(syntax: UnexpectedNodesSyntax) -> UnexpectedNodesSyntax {
     syntax
   }
-  func format(syntax: InOutExprSyntax) -> InOutExprSyntax {
+  public func format(syntax: InOutExprSyntax) -> InOutExprSyntax {
     var result = syntax
     let leadingTrivia = result.leadingTrivia ?? []
     if !leadingTrivia.isEmpty {
       result = result.withLeadingTrivia(leadingTrivia.addingSpacingAfterNewlinesIfNeeded())
     }
     return result
   }
-  func format(syntax: PoundColumnExprSyntax) -> PoundColumnExprSyntax {
+  public func format(syntax: PoundColumnExprSyntax) -> PoundColumnExprSyntax {
     var result = syntax
     let leadingTrivia = result.leadingTrivia ?? []
     if !leadingTrivia.isEmpty {
       result = result.withLeadingTrivia(leadingTrivia.addingSpacingAfterNewlinesIfNeeded())
     }
     return result
   }
-  func format(syntax: TupleExprElementListSyntax) -> TupleExprElementListSyntax {
+  public func format(syntax: TupleExprElementListSyntax) -> TupleExprElementListSyntax {
     syntax
   }
-  func format(syntax: ArrayElementListSyntax) -> ArrayElementListSyntax {
+  public func format(syntax: ArrayElementListSyntax) -> ArrayElementListSyntax {
     syntax
   }
-  func format(syntax: DictionaryElementListSyntax) -> DictionaryElementListSyntax {
+  public func format(syntax: DictionaryElementListSyntax) -> DictionaryElementListSyntax {
     syntax
   }
-  func format(syntax: StringLiteralSegmentsSyntax) -> StringLiteralSegmentsSyntax {
+  public func format(syntax: StringLiteralSegmentsSyntax) -> StringLiteralSegmentsSyntax {
     syntax
   }
-  func format(syntax: TryExprSyntax) -> TryExprSyntax {
+  public func format(syntax: TryExprSyntax) -> TryExprSyntax {
     var result = syntax
     let leadingTrivia = result.leadingTrivia ?? []
     if !leadingTrivia.isEmpty {
       result = result.withLeadingTrivia(leadingTrivia.addingSpacingAfterNewlinesIfNeeded())
     }
     return result
   }
-  func format(syntax: AwaitExprSyntax) -> AwaitExprSyntax {
+  public func format(syntax: AwaitExprSyntax) -> AwaitExprSyntax {
     var result = syntax
     let leadingTrivia = result.leadingTrivia ?? []
     if !leadingTrivia.isEmpty {
       result = result.withLeadingTrivia(leadingTrivia.addingSpacingAfterNewlinesIfNeeded())
     }
     return result
   }
-  func format(syntax: MoveExprSyntax) -> MoveExprSyntax {
+  public func format(syntax: MoveExprSyntax) -> MoveExprSyntax {
     var result = syntax
     let leadingTrivia = result.leadingTrivia ?? []
     if !leadingTrivia.isEmpty {
       result = result.withLeadingTrivia(leadingTrivia.addingSpacingAfterNewlinesIfNeeded())
     }
     return result
   }
-  func format(syntax: DeclNameArgumentSyntax) -> DeclNameArgumentSyntax {
+  public func format(syntax: DeclNameArgumentSyntax) -> DeclNameArgumentSyntax {
     var result = syntax
```

Comment:
This code is so repetitive. Can we not put it into an extension of `SyntaxProtocol` somehow instead of stamping out a hundred copies?

Reply:
Could definitely be something we can investigate in a follow-up PR.